pycbc/waveform/bank.py

```diff
@@ -68,6 +68,11 @@ class BaseFilterBank(object):
     about elements of an xml template bank.
     """
     def __init__(self, filename, approximant=None, **kwds):
+
```

Comment:
Knowing a bit more now I'm currently making another attempt at doing the input file handling without any modifications to PyCBC, and without that patch in particular.